agents/plugins/apache_status

```diff
@@ -201,15 +201,15 @@ def main():
 
         try:
             response = get_response(proto, cafile, address, portspec, page)
-            for line in response.read().split('\n'):
+            for line in response.read().splitlines():
                 if not line.strip():
                     continue
                 if line.lstrip()[0] == '<':
                     # Seems to be html output. Skip this server.
                     break
                 if not name:
                     name = get_instance_name(address, port, get_instance_name_map(config))
-                sys.stdout.write("%s|%s|%s|%s\n" % (address, port, name, line))
+                sys.stdout.write("%s|%s|%s|%s\n" % (address, port, name, line.encode('utf-8'))
```

Comment:
This is not correct: The type of `sys.stdout` under Python 2 is `IO[str]`, under Python 3 it is `TextIO` (i.e. effectively `IO[str]` again). But `str` is not `str`. :wink: In other words: Regardless of the Python version, `sys.stdout` should be used with `str` only.